chain-spec-generator/src/system_parachains_specs.rs

```diff
@@ -0,0 +1,165 @@
+		(get_account_id_from_seed::<sr25519::Public>("Bob"), get_from_seed::<AuraId>("Bob")),
+	]
+}
+
+/// Invulnerable Collators for the particular case of AssetHubPolkadot
+pub fn invulnerables_asset_hub_paseo() -> Vec<(AccountId, AssetHubPolkadotAuraId)> {
+	vec![
+		(
+			get_account_id_from_seed::<sr25519::Public>("Alice"),
+			get_from_seed::<AssetHubPolkadotAuraId>("Alice"),
+		),
+		(
+			get_account_id_from_seed::<sr25519::Public>("Bob"),
+			get_from_seed::<AssetHubPolkadotAuraId>("Bob"),
+		),
+	]
+}
+
+/// Generate the session keys from individual elements.
+///
+/// The input must be a tuple of individual keys (a single arg for now since we have just one key).
+pub fn asset_hub_paseo_session_keys(
+	keys: AssetHubPolkadotAuraId,
+) -> asset_hub_paseo_runtime::SessionKeys {
+	asset_hub_paseo_runtime::SessionKeys { aura: keys }
+}
+
+
+// AssetHubPolkadot
+fn asset_hub_paseo_genesis(
+	wasm_binary: &[u8],
+	invulnerables: Vec<(AccountId, AssetHubPolkadotAuraId)>,
+	endowed_accounts: Vec<AccountId>,
+	id: ParaId,
+) -> asset_hub_paseo_runtime::RuntimeGenesisConfig {
+	asset_hub_paseo_runtime::RuntimeGenesisConfig {
+		system: asset_hub_paseo_runtime::SystemConfig {
+			code: wasm_binary.to_vec(),
+			..Default::default()
+		},
+		balances: asset_hub_paseo_runtime::BalancesConfig {
+			balances: endowed_accounts
+				.iter()
+				.cloned()
+				.map(|k| (k, ASSET_HUB_PASEO_ED * 4096))
+				.collect(),
+		},
+		parachain_info: asset_hub_paseo_runtime::ParachainInfoConfig {
+			parachain_id: id,
+			..Default::default()
+		},
+		collator_selection: asset_hub_paseo_runtime::CollatorSelectionConfig {
+			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
+			candidacy_bond: ASSET_HUB_PASEO_ED * 16,
+			..Default::default()
+		},
+		session: asset_hub_paseo_runtime::SessionConfig {
+			keys: invulnerables
+				.into_iter()
+				.map(|(acc, aura)| {
+					(
+						acc.clone(),                           // account id
+						acc,                                   // validator id
+						asset_hub_paseo_session_keys(aura), // session keys
+					)
+				})
+				.collect(),
+		},
+		// no need to pass anything to aura, in fact it will panic if we do. Session will take care
+		// of this.
+		aura: Default::default(),
+		aura_ext: Default::default(),
+		parachain_system: Default::default(),
+		polkadot_xcm: asset_hub_paseo_runtime::PolkadotXcmConfig {
+			safe_xcm_version: Some(SAFE_XCM_VERSION),
+			..Default::default()
+		},
+	}
+}
+
+fn asset_hub_paseo_local_genesis(
+	wasm_binary: &[u8],
+) -> asset_hub_paseo_runtime::RuntimeGenesisConfig {
+	asset_hub_paseo_genesis(
+		// initial collators.
+		wasm_binary,
+		invulnerables_asset_hub_paseo(),
+		testnet_accounts(),
+		1000.into(),
+	)
+}
+
+pub fn asset_hub_paseo_local_testnet_config() -> Result<Box<dyn ChainSpec>, String> {
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("ss58Format".into(), 0.into());
```

Comment:
This should be 42 I'd say
```suggestion
	properties.insert("ss58Format".into(), 42.into());
```